Extensions/Effects/JsExtension.js

```diff
@@ -138,12 +138,115 @@ module.exports = {
       .setDescription(_('Alter the colors to sepia'))
       .addIncludeFile('Extensions/Effects/sepia-pixi-filter.js');
     const sepiaProperties = sepiaEffect.getProperties();
-    sepiaProperties.set(
-      'opacity',
-      new gd.PropertyDescriptor(/* defaultValue= */ '1')
-        .setLabel(_('Opacity (between 0 and 1)'))
-        .setType('number')
-    );
+    sepiaProperties.set('opacity', new gd.PropertyDescriptor(/* defaultValue= */ '1').setLabel(_('Opacity (between 0 and 1)')).setType('number'));
```

Comment:
Please run Prettier on the file :) 
I know that you would prefer to have everything on a single line, but auto-formatting is giving 10x more benefits than formatting manually.

Also I understand the interest for "one liners". "It's a single line it's shorter!" I used to like this :) But if you have multiple calls to functions, there is no need to "hide" them on a line. It's on the contrary very clear:
...
.setSomething(toThis)
.setThis(toThat)
.setThat(toThis)

Reply:
At some point, we can't reduce complexity. If our filters need parameters, we need lines for them :) It's not a problem. It's what effects are!

Reply:
I have prettier running now, but it updated some lines that are not a part of the pr, so I wonder if my prettier settings are not aligned. Do we have something in the readme about how to set prettier?

Reply:
Are you sure you've run Prettier and not some kind of default autoformatting tool like the one bundled in VS Code?

Reply:
Otherwise I can do it, let me know